docs/verifiable-document/document-template.md

```diff
@@ -4,8 +4,285 @@ title: Deploying Document Renderer
 sidebar_label: Deploying Document Renderer
 ---
 
-Overview
+OA documents are both readable by machines as well as humans. Every OA document file is stored in `.json` format, allowing any applications to process the content within. To present the data file in a human readable format, a renderer need to be written to display the data in a familiar format to humans. This renderer is a static website that will be embedded in compliant OA viewer.
```

Comment:
- as well as humans. => as well as BY humans.
- a renderer need => a renderer needs.
- To present the data file in a human readable format, a renderer need to be written to display the data in a familiar format to humans. => that's very redundant, I would remove one part of the sentence